Doc/library/plistlib.rst

```diff
@@ -130,7 +133,7 @@ The following classes are available:
 .. class:: UID(data)
 
    Wraps an :class:`int`.  This is used when reading or writing NSKeyedArchiver
-   encoded data, which contains UID (see PList manual).
+   encoded data, which contains UID (see Wikipedia).
```

Comment:
It's not preferable to refer to Wikipedia.

Reply:
I know, but Apple doesn't document the archivers. Should I just link to the Swift foundation source?

Reply:
```suggestion
   encoded data, which contains UID (see Swift foundation source).
```